addons/payment/tests/test_payment_transaction.py

```diff
@@ -80,3 +84,21 @@ def test_refund_transaction_values(self):
             msg="The amount of the refund transaction should be the negative value of the amount "
                 "to refund."
         )
+
+    @mute_logger('odoo.addons.payment.models.payment_transaction')
+    def test_update_state_from_wrong_state(self):
+        tx = self._create_transaction('redirect', state='done')
+        self.assertFalse(
+            tx._update_state(allowed_states=['draft', 'pending', 'authorized'], target_state='cancel', extra_allowed_states=None, state_message=None),
+            msg="Transaction without extra allowes states should not be able to change state"
+                "if in wrong state'"
+            )
+
+    def test_update_with_extra_allowed_state(self):
```

Comment:
```suggestion
    def test_update_state_with_extra_allowed_state(self):
```
Put the whole name of the method that is tested

Reply:
Okay :)